spec/requests/api/v3/user/create_user_resource_spec.rb

```diff
@@ -0,0 +1,208 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+
+require 'spec_helper'
+require 'rack/test'
+
+describe ::API::V3::Users::UsersAPI do
+  include Rack::Test::Methods
+  include API::V3::Utilities::PathHelper
+
+  let(:path) { api_v3_paths.users }
+  let(:user) { FactoryGirl.build(:admin) }
+  let(:parameters) { {} }
+
+  before do
+    login_as(user)
+  end
+
+  def send_request
+    post path, parameters.to_json, 'CONTENT_TYPE' => 'application/json'
```

Comment:
Shouldn't this be `post path, params: parameters.to_json, headers: { "Content-Type": "application/json" }`?
At least I use that and it works. Do you not get any deprecation warning for this?